datadotworld/client/swagger-dwapi-def.json

```diff
@@ -0,0 +1,1400 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "0.1-SNAPSHOT",
+    "title": "data.world Public API"
+  },
+  "host": "api.data.world",
+  "basePath": "/v0",
+  "tags": [
+    {
+      "name": "datasets"
+    },
+    {
+      "name": "uploads"
+    }
+  ],
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/datasets/{owner}": {
+      "post": {
+        "tags": [
+          "datasets"
+        ],
+        "summary": "Create a new dataset.",
+        "description": "",
+        "operationId": "createDataset",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "owner",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/DatasetCreateRequest"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Dataset created successfully.",
+            "schema": {
+              "$ref": "#/definitions/SuccessMessage"
+            }
+          },
+          "202": {
```

Comment:
That change is inconsequential (doc aside). I'll re-generate the swagger client when we have new API functionality to use.